.eslintrc.json

```diff
@@ -108,6 +108,9 @@
         },
         {
             "files": ["*.ts", "*.tsx"],
+            "parserOptions": {
```

Comment:
This is definetly not correct. Setting parserOptions.project in the root will make nx parse the entire repo for any lint operation. https://nx.dev/recipes/other/eslint#rules-requiring-type-information

Reply:
ah, makes sense. My bad